deployment-examples/aws/serverless/extension/README.md

```diff
@@ -7,7 +7,8 @@ You could use a deployment package to deploy your function code to Lambda.
 To bundle the extension module, navigate into `extension` folder and run `npm run zip-lambda-function`. 
 4. Specify `index.handler` as the entry point for the AWS Lambda function.
     > The default value in the console is index.handler which calls exports.handler in index.js
-5. Please check our [faq](../../../../docs/FAQ.md) for your deployment.
+5. Use version 2.0 of `Payload format version` in the AWS API Gateway.
```

Comment:
It looks more specific to AWS API Gateway settings,
As mentioned here in the [docs](https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-lambda.html), The default version is always the latest version.

Do you see any problem with the Lambda function here? Did you find issues using version 1.0 ?